packages/@jsii/go-runtime/jsii-runtime-go/kernel/client.go

```diff
@@ -37,8 +39,8 @@ type client struct {
 	stdin  io.WriteCloser
 	tmpdir string
 
-	types   typeregistry.TypeRegistry
-	objects map[reflect.Value]string
+	types   *typeregistry.TypeRegistry
+	objects *objectstore.ObjectStore
```

Comment:
Re-factored into separate module, as keying on `reflect.Value` resulted in bugs, because they compare `==` if their values are `==`, regardless of whether they are distinct object instances... But this map actually should track individual instances (i.e: pointers).